### ISSUE TITLE
Daily meeting (20/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Gallery-Shopping-Website#9]
So how was the progress?

**Comment by kcchan3012** [comment]
So I just designed the layout for the add to cart and it contain few box such as name, quantity, remove. Now I working on the behind code.

**Comment by boonsenglim** [comment]
For me i just finished design for the payment layout part, and now i designing the thank you page .

**Comment by pqyyieeuyi** [comment]
I take boon seng layout and now doing the validation for the text box. I have a question how to validate the user visa or master card number are correct?

**Comment by boonsenglim** [comment]
Erm, for that I think you should validate the length of the card number?

**Comment by kcchan3012** [comment]
Not only length, you need to validate the first number of the card number such as visa is "5" and master is "4".

**Comment by Lai0723** [comment]
I disagree with you kar chen, actually the first number of the Visa card  is "4" not "5". Boon seng did you use the radio button for the user to choose the payment type?

**Comment by boonsenglim** [comment]
Ya, I did but why you ask that?

**Comment by Lai0723** [comment]
Because in that case benjamin you can write a code that to validate user is using master or visa.

**Comment by kcchan3012** [comment]
Can you further explain it?

**Comment by pqyyieeuyi** [comment]
Maybe like this when user choose the visa radio button the validation for the card number should be "4" and "5" for the user that choose master card radio button?

**Comment by Lai0723** [comment]
Ya, this is what I mean

**Comment by boonsenglim** [comment]
Then you should use if else case for that.

**Comment by pqyyieeuyi** [comment]
Ya I notice that, by the way thanks for the idea.

**Comment by Lai0723** [comment]
Anymore problem so far?

**Comment by kcchan3012** [comment]
Since I still working on the coding and no problem so far.

**Comment by boonsenglim** [comment]
I still working on the thank you page and no problem so far to.

**Comment by pqyyieeuyi** [comment]
I think i just asked i wanted to know and now i will go try do the validation.

**Comment by Lai0723** [comment]
If no problem I think that's all for today meeting, everyone can dismiss now.
